src/app/collection/config/collection-config.component.html

```diff
@@ -37,8 +37,8 @@
          class="count"
          title="Number of documents in this collection"
          [style.color]="collection.color"
-         [routerLink]="[workspacePath(), 'view']"
-         [queryParams]="{ perspective: 'postit', query: documentsQuery(collection.code) }">
+         [routerLink]="[workspacePath(), 'view', 'postit']"
```

Comment:
We need this change also in `search-home.component.html`. :)

Reply:
It has already been fixed in commit 05db543960d2bf32d429e80bb50c9357365b5075.